### PR TITLE
Bugfix unable to create user 

### PR DESCRIPTION
**Pull request by stlgaits**
Since adding timezone to User Entity, the commands to create a new Myddleware User weren't working anymore, due to missing code to add user timezone to the new object.
This has been fixed in both methods (Add User Command from a terminal + UI Form on /register path).
I've also added code in case there's an error on the UI /register form submit.
This merge also includes an upgrade of some packages, moving on to Symfony 4.4.37.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ Encore
     // directory where compiled assets will be stored
     .setOutputPath('public/build/')
     // public path used by the web server to access the output path
-    .setPublicPath('./build')
+    .setPublicPath('/build')
```
Comment (Myddleware) [inline webpack.config.js:13]: @EstelleMyddleware , are you sure it is safe to push this change to main branch ?

**Review by Myddleware — COMMENTED** [review]
Please @EstelleMyddleware  could you also change the Myddleware version here, should be 3.0.5b : https://github.com/Myddleware/myddleware/blob/6e2da2b61dfe1872358381183e679c935dcb749e/.env#L1

Thanks.

File: composer.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
   "prefer-stable": true,
     "require": {
         "php": ">=7.4",
-        "symfony/symfony": "^4.4.30",
+        "symfony/symfony": "^4.4.37",
```
Comment (Myddleware) [inline composer.json:17]: @EstelleMyddleware , we should avoid to upgrade Symfony on a hotfix version if it is possible. 
If we have to upgrade Symfony, we should request non regression tests from @RomanFaure .

**Review by Myddleware — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (stlgaits) [reply composer.json:17]: @Myddleware Normally it should be safe to upgrade as it is only a minor version https://symfony.com/doc/current/contributing/community/releases.html#backward-compatibility

**Pull request closed without merging**